### ISSUE TITLE
createAcc - отлов дубликатов

### ISSUE THREAD

**Issue by elagin** [rjhdby/MobileServer#7]
На вход добавить необязательный параметр bool createOwerride, который по умолчанию не передается.

Если он не передан, то проверять на сервере существующие (включая скрытые) и если найдено недалеко, то предупреждать об этом пользователя включайя текст аналогичный для "поделиться".

Если он присутствует - создавать без проверки.

**Comment by rjhdby** [comment]
Нифига не понял

**Comment by elagin** [comment]
1) Если во входных параметрах есть createOwerride=true - старое поведение (создаем новый инцидент).

2) Если во входных параметрах нет такого параметра - поверяем нет ли рядом "похожего" инцидента.
а) Инцидента нет - старое поведение (создаем новый инцидент).
б) Инцидент уже есть, то не создаем, а возвращаем already_exists="Прочее, Москва, Красная площадь, Топливо".

"похожий" - хотя бы рядом расположенный, а так же, вероятно и по типу совпадающий.

**Comment by rjhdby** [comment]
Как определяем похожесть?

**Comment by elagin** [comment]
См. последнее предложение в последнем комметарии.

**Comment by elagin** [comment]
Наша задача обнаружить **возможный** дубликат и сообщить об этом автору, а уж он путь подумает пытался ли он дубликат создать.

**Comment by rjhdby** [comment]
Бывали случаи двух дтп примерно рядом в примерно то-же самое время.

В общем нетривиальная задача, надо все тщательно продумать.

**Comment by elagin** [comment]
Да, бывает всякое, путь это и решает создатель второго инцидента.Наша первая задача подсказать ему, что уже есть рядом инцидент.
